### ISSUE TITLE
dotnet-core-uninstall might consider the INVERSE of --dry-run

### ISSUE THREAD

**Issue by shanselman** [dotnet/cli-lab#89]
PowerShell uses --whatif, by the way, as a standard.

We might consider /force or /doit. There's too much risk to "Accidentally" hurt myself.

**Comment by KathleenDollard** [comment]
Let's change from options to verbs to indicate what will be done:

```
dotnet-core-uninstall list
dotnet-core-uninstall dry-run
dotnet-core-uninstall what
dotnet-core-uninstall remove
```

`dry-run` and `whatif` would be synonyms. 

I hate that one is posix and the other isn't, but if we include `whatif` it should be just like PowerShell

I believe this answers @shanselman's concern by avoiding a destructive default.

**Comment by sfoslund** [comment]
Addressed in Restructure Commands PR (#96 )
